packages/nextjs/services/web3/stark-burner/BurnerConnector.ts

```diff
@@ -69,6 +87,39 @@ export class BurnerConnector extends InjectedConnector {
     return Promise.resolve((await this.account()).address !== "");
   }
 
+  async request<T extends RpcMessage["type"]>(
+    call: RequestFnCall<T>,
+  ): Promise<RpcTypeToMessageMap[T]["result"]> {
+    if (call.params && "calls" in call.params) {
+      let compiledCalls = call.params.calls;
+      try {
+        // TODO : starknet connector uses "emtrypoint" instead of "entry_point"
+        // TODO : starknet connector uses "contract_address" instead of "contractAddress"
+        compiledCalls.forEach((element) => {
```

Comment:
this was also a temporary solution,  and the description was mentioned on the TODOs comments , lets check if this is still the case.

Reply:
this one hard to test, how did you log this? I tried console logging and even stdout but doesnt print anything

Reply:
when not doing this step the transaction will fail validation, the calldata will not be parsed because is missing due to different naming , check [here](https://github.com/apibara/starknet-react/blob/56f352dd44a8a746a25536106f712e9e1f673da6/packages/core/src/hooks/use-send-transaction.ts#L75C1-L84C2)